packages/smart-contracts/scripts-create2/compute-from-list.ts

```diff
@@ -0,0 +1,38 @@
+import { computeCreate2DeploymentAddress } from './compute-one-address';
+import { create2ContractDeploymentList } from './utils';
+import { HardhatRuntimeEnvironmentExtended } from './types';
+
+export const computeCreate2DeploymentAddressesFromList = async (
+  hre: HardhatRuntimeEnvironmentExtended,
+): Promise<void> => {
+  if (!hre.config.xdeploy.networks || !hre.config.xdeploy.rpcUrls) {
+    throw new Error('Bad network configuration');
+  }
+
+  if (!hre.config.xdeploy.salt) {
+    throw new Error('Missing salt');
+  }
+
+  if (!hre.config.xdeploy.deployerAddress) {
+    console.warn('Deployer address is set to default !');
+  }
+
+  await Promise.all(
+    create2ContractDeploymentList.map(async (contract) => {
+      let address: string;
+      switch (contract) {
+        case 'EthereumProxy':
+          address = await computeCreate2DeploymentAddress({ contract: 'EthereumProxy' }, hre);
+          console.log(`EthereumProxy                ${address}`);
+          break;
+        case 'EthereumFeeProxy':
+          address = await computeCreate2DeploymentAddress({ contract: 'EthereumFeeProxy' }, hre);
+          console.log(`EthereumFeeProxy             ${address}`);
```

Comment:
```suggestion
        case 'EthereumFeeProxy':
          address = await computeCreate2DeploymentAddress({ contract }, hre);
          console.log(`${contract}   ${address}`);
```
+ I think there is a util for text alignment in the former deployment script.